bot.go

```diff
@@ -320,7 +321,7 @@ func errorsBlock(errs []string) string {
 		}
 		res += "\n-\t" + strings.Join(strings.Split(e, "\n"), "\n-\t")
 	}
-	res += "\n```"
+	res += "\n``` _Please **edit** your message to correct instead of making a new one (less floody that way). Thank you!_"
```

Comment:
We know people are a pain🤭, but usually we don't tell them, they dislike facing the truth 😅😂

What about this?


```suggestion
	res += "\n``` _Tips: You can **edit** your message to correct instead of making a new one. It also avoids extra messages_"
```

Or anything like that

Reply:
so I have the Tip version in the help (that nobody reads) but if there is an error I figured it's a chance to... gently encourage... but yeah maybe I can soften it here too

Reply:
<img width="452" alt="Screenshot 2024-08-27 at 7 29 06 PM" src="https://github.com/user-attachments/assets/28d2bef4-6019-4c45-9958-e4340e9c8f00">

Reply:
<img width="792" alt="Screenshot 2024-08-28 at 10 05 03 AM" src="https://github.com/user-attachments/assets/c9a2269f-8271-40c0-a073-58f4cbba821c">

Reply:
shortening a bit
```
	res += "\n```Tip: _You can **edit** your message to correct instead of making a new one!_"
```
because instead of making a new one... implies it avoids extra messages